### DISCUSSION TITLE
Capital raised for my project

### DISCUSSION THREAD

**Discussion by the-thaqalayn** [community/community#66975] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How can I capital raised for my project on GitHub?

**Comment by mehradi-github** [comment]
You can become a sponsored developer by joining GitHub Sponsors, completing your sponsored developer profile, creating sponsorship tiers, submitting your bank and tax information, and enabling two-factor authentication for your account on GitHub.com.
- Joining GitHub Sponsors
- Completing your sponsored developer profile
- Creating sponsorship tiers
- Submitting your bank information
- Submitting your tax information
- Enabling two-factor authentication (2FA) on your GitHub account
- Submitting your application to GitHub for approval

For more details you can see [this link](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account#creating-sponsorship-tiers).

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
